### PR TITLE
Mobile design

### PR DESCRIPTION
**Pull request by bludojr**

### CODE REVIEW

File: src/components/BookHook.jsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+
+const BookHook = () => {
+  return (
+    <>
+        <div className='flex space-x-[16px] w-[304] ml-[12px] h-[48px] ' >
```
Comment (amalisaah) [inline src/components/BookHook.jsx:6]: you are using tailwind but you are keeping to setting arbitrary values for everything even the ones handled by tailwind

File: src/components/BookHook.jsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+
+const BookHook = () => {
```
Comment (amalisaah) [inline src/components/BookHook.jsx:3]: it seems to me like you repeated the same div over and over again

Comment (amalisaah) [reply src/components/BookHook.jsx:3]: image

[https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/from](Array.from)

[https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map](map)

Comment (amalisaah) [reply src/components/BookHook.jsx:3]: for now you can do this 

image

**Review by amalisaah — CHANGES_REQUESTED** [review]

File: src/components/Background.jsx [hunk 3]
```diff
@@ -1,17 +1,24 @@
...
-          className=" min-w-[360px] w-[360px] h-[800px] border-white border-[12px] rounded-3xl overflow-hidden mx-auto mt-16 bg-[linear-gradient(192.04deg, #5E54F7 0%, #F7A5CB 54.14%, #C47151 100%)]"
-        ></div>
+          className=" min-w-96 w-96 h-[800px] border-white border-[12px] rounded-3xl overflow-hidden mx-auto mt-16 bg-[linear-gradient(192.04deg,_#5E54F7_0%,_#F7A5CB_54.14%,_#C47151_100%)] backdrop-blur-lg"
+        >
+          <Card />
+          <TodoFrame/>
+        </div>
       </div>
-      <div className="w-[497px] h-[497px] rounded-full blur-3xl bg-[#00F0FF1A] absolute bottom-[-200px] right-[-200px]"></div>
+      <div className="w-[497px] h-[497px] rounded-full blur-3xl bg-[#00F0FF1A] relative bottom-[-200px] right-[-200px]"></div>
```
Comment (amalisaah) [inline src/components/Background.jsx]: is there a reason for setting the position?

File: src/components/BookHook.jsx
Comment (amalisaah) [inline src/components/BookHook.jsx:1]: please update this file

File: src/components/Card.jsx [hunk 4]
```diff
@@ -1,8 +1,25 @@
...
+import NotificationArea from './NotificationArea'
+import Panel from "./Panel";
+
+
 const Card = () => {
   return (
     <>
+      <div className="">
+        <div className="w-[360px] h-[80px] border-t-0 border-l-0 border-r-0 rounded-b-[12px] bg-gradient-to-l from-[#8378eb] to-[#cdbaf1] border-[#a794ea] border-[1px] flex items-center shadow-[14.65px 17.58px 29.29px 0px #0000000D] backdrop-blur-[20.5px]" 
+             style={{
```
Comment (amalisaah) [inline src/components/Card.jsx]: remove this line

**Review by amalisaah — CHANGES_REQUESTED** [review]

File: src/components/Background.jsx (hunk 3 above)
Comment (bludojr) [reply src/components/Background.jsx]: They are backdrop blur effects which where annotated in the figma design
I have replaced the arbitrary values with custom configurations

File: src/components/BookHook.jsx
Comment (bludojr) [reply src/components/BookHook.jsx:1]: I have replaced the arbitrary values with custom configurations

File: src/components/Card.jsx (hunk 4 above)
Comment (bludojr) [reply src/components/Card.jsx]: I have removed the in-line styling line of code

File: src/components/Background.jsx [hunk 5]
```diff
@@ -6,16 +6,16 @@ const Background = () => {
   return (
     <>
       
-      <div className="w-[497px] h-[497px] rounded-full blur-3xl bg-[#4400FF1F] absolute left-[-200px] top-[-200px]"></div>
+      <div className="w-custom1 h-custom1 rounded-full blur-3xl bg-custom1 absolute -left-50 -top-50 "></div>
```
Comment (bludojr) [inline src/components/Background.jsx:9]: I have replaced the arbitrary values with custom configurations

File: src/components/Background.jsx [hunk 6]
```diff
@@ -6,16 +6,16 @@ const Background = () => {
...
       <div className="h-screen w-screen bg-transparent">
         <div
-          className=" min-w-96 w-96 h-[800px] border-white border-[12px] rounded-3xl overflow-hidden mx-auto mt-16 bg-[linear-gradient(192.04deg,_#5E54F7_0%,_#F7A5CB_54.14%,_#C47151_100%)] backdrop-blur-lg"
+          className=" min-w-96 w-96 h-200 border-white border-12 rounded-3xl overflow-hidden mx-auto mt-16 bg-custom-gradient backdrop-blur-lg"
```
Comment (bludojr) [inline src/components/Background.jsx:12]: I have replaced the arbitrary values with custom configurations

File: src/components/Background.jsx [hunk 7]
```diff
@@ -6,16 +6,16 @@ const Background = () => {
...
         >
           <Card />
           <TodoFrame/>
         </div>
       </div>
-      <div className="w-[497px] h-[497px] rounded-full blur-3xl bg-[#00F0FF1A] relative bottom-[-200px] right-[-200px]"></div>
+      <div className="w-custom1 h-custom1 rounded-full blur-3xl bg-custom2 relative -bottom-50 -right-50"></div>
```
Comment (bludojr) [inline src/components/Background.jsx:18]: I have replaced the arbitrary values with custom configurations

File: src/components/Card.jsx [hunk 8]
```diff
@@ -7,10 +7,7 @@ const Card = () => {
   return (
     <>
       <div className="">
-        <div className="w-[360px] h-[80px] border-t-0 border-l-0 border-r-0 rounded-b-[12px] bg-gradient-to-l from-[#8378eb] to-[#cdbaf1] border-[#a794ea] border-[1px] flex items-center shadow-[14.65px 17.58px 29.29px 0px #0000000D] backdrop-blur-[20.5px]" 
-             style={{
-             
-             }}
+        <div className="w-90 h-20 border-t-0 border-l-0 border-r-0 rounded-b-12 bg-gradient-to-l from-[#8378eb] to-[#cdbaf1] border-[#a794ea] border-b flex items-center shadow-[14.65px 17.58px 29.29px 0px #0000000D] backdrop-blur-[20.5px]"
```
Comment (bludojr) [inline src/components/Card.jsx:10]: I have replaced the arbitrary values with custom configurations

File: src/components/NotificationArea.jsx [hunk 9]
```diff
@@ -2,7 +2,7 @@ import React from "react";
 
 const NotificationArea = () => {
   return (
-    <div className="mt-[-53px] ml-[16px] text-white" >
+    <div className="-mt-13.25 ml-4 text-white" >
```
Comment (bludojr) [inline src/components/NotificationArea.jsx:5]: I have replaced the arbitrary values with custom configurations

File: tailwind.config.js [hunk 10]
```diff
@@ -1,14 +1,43 @@
...
+      width: {
+        custom1: "497px",
+        "200": "50rem",
+        "90": "22.5rem"
+      },
       fontFamily: {
-        inter: ['inter', 'sans-serif'],
-      }
+        "inter": ["inter", "sans-serif"],
+      },
```
Comment (bludojr) [inline tailwind.config.js:40]: Added custom configurations for tailwind styling

**Review by amalisaah — APPROVED** [review]

**Pull request merged**